core/utils.py

```diff
@@ -0,0 +1,19 @@
+def match_stack_number(i):
+    if i == 1:
+        return "프론트"
+    if i == 2:
+        return "백"
+    else:
+        return ""
+
+def match_alcohol_limit_number(i):
+    if i == 1:
+        return "알쓰"
+    if i == 2:
+        return "반병"
+    if i == 3:
+        return "한병"
+    if i == 4:
+        return "N병"
+    else:
+        return ""
```

Comment:
if문을 사용하지 않고 dictionary로 만들어주세요.
```suggestion
     t = { }
     ..
```
로직은 선형성을 갖도록 만들어주시는 것이 좋습니다. if문을 많이 사용하는 경우에는 그만큼 로직이 갈라지기 때문에 많은 if은 dictionary로 변경하므로써 로직을 선형적으로 만들어줄 수 있습니다.

Reply:
딕셔너리로 바꾸어 수정하였습니다.